src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -450,14 +450,13 @@ private AcceptTxResult AddCore(Transaction tx, TxFilteringState state, bool isPe
             if (tx.Hash == removed?.Hash)
             {
                 // it means it was added and immediately evicted - pool was full of better txs
-                if (isPersistentBroadcast)
+                if (!isPersistentBroadcast || tx.SupportsBlobs)
                 {
-                    // we are adding only to persistent broadcast - not good enough for standard pool,
-                    // but can be good enough for TxBroadcaster pool - for local txs only
-                    _broadcaster.Broadcast(tx, isPersistentBroadcast);
```

Comment:
what happened to ``_broadcaster.Broadcast`` call?

Reply:
Since the if branch now has the opposite condition, if the `tx` is persistent (and not a blob tx), it would run to the end of the function (line 462~483).

`_broadcaster.Broadcast` will be run at line 477.

Reply:
Not sure if running through all the normal flow here is best choice as this is a special case, will wait for @marcindsobczak opinion.